scripts/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -17,6 +18,9 @@ public class PrepareTargets
         [Target(nameof(Init), nameof(RestorePackages))]
         public static BuildTargetResult Prepare(BuildTargetContext c) => c.Success();
 
+        [Target(nameof(CheckPrereqCmakePresent), nameof(CheckPrereqDebianPackageBuildComponents), nameof(CheckPrereqCoreclrDependencyPackages))]
```

Comment:
I'm not sure about this factoring.

CheckPlatformSpecificBuildComponents seems more appropriate. That can fork based on the active platform.